src/tests/functional/plugin/shared/src/behavior/plugin/configuration_tests.cpp

```diff
@@ -192,17 +192,20 @@ TEST_P(ExclusiveAsyncReqTests, excluAsyncReqTests) {
     ASSERT_NO_THROW(ie->LoadNetwork(cnnNet, target_device, configuration));
 }
 
-TEST_P(SetPropLoadNetWorkGetPropTests, SetPropLoadNetWorkGetProperty) {
+TEST_P(SetPropLoadNetWorkGetPropTests, ieSetGetConfigAndLoadNetWorkSetGetPropertyTest) {
     ASSERT_NO_THROW(ie->SetConfig(configuration, target_device));
 
     InferenceEngine::ExecutableNetwork exeNetWork;
     ASSERT_NO_THROW(exeNetWork = ie->LoadNetwork(cnnNet, target_device, loadNetWorkConfig));
-    if (target_device == CommonTestUtils::DEVICE_AUTO)
-        GTEST_SKIP();
 
+    //ie's setConfig and LoadNetwork should not affect each other, for config settings
     for (const auto& property_item : loadNetWorkConfig) {
         InferenceEngine::Parameter exeNetProperty;
-        ASSERT_NO_THROW(exeNetProperty = exeNetWork.GetConfig(property_item.first));
+        try {
```

Comment:
The same: why do you change such construction?

Reply:
Adopted from the implementation of CompiledModel::get_property(), if GetConfig fails and then try GetMetric. ASSERT_NO_THROW on GetMetric

Reply:
If AUTO/MULTI can't support this test case, skip it in skip file and with a JIRA ticket to follow up.

Reply:
TODO: CVS-104942